test/correctness/fuzz_simplify.cpp

```diff
@@ -242,61 +255,6 @@ bool test_expression(Expr test, int samples) {
     return true;
 }
 
-Expr ramp(Expr b, Expr s, int w) {
```

Comment:
FYI, these were here to make it possible to copy and paste failing expressions from failure outputs into the test to debug them (in place of randomly generated expressions). 

(There should have been a comment on them to explain this...)

Reply:
Ah, that's why! Sorry about that, restored them back.

Reply:
Can you also add a comment that explains them?

// These are here to enable copy failed output expressions and pasting them into the test for debugging.

Reply:
Done